uiowa/uiowa
# [pharmacy.uiowa.edu] Territory updates

Resolves #6485 





# How to test


`ddev blt ds --site=pharmacy.uiowa.edu && ddev drush @pharmacy.local uli`

Update hook should complete cleanly, followed by config import with no errors.

Edit a few existing Persons with the "Palliative Care Graduate" person type. 
Compare with their prod counterparts and check that they still have the same state(s) selected.
Check that there is no "International" option

Add a new Person with the "Palliative Care Graduate" person type.
Check that there is no "International" option

Check out https://pharmacy.uiowa.ddev.site/pharmd/dual_cert/palliative/certificate_graduates and see that there is no "International" option (as opposed to https://pharmacy.uiowa.edu/pharmd/dual_cert/palliative/certificate_graduates)

Extra credit testing option:
`git checkout main && ddev blt ds --site=pharmacy.uiowa.edu && ddev drush @pharmacy.local uli`
Edit a person and select the International option
`git checkout pharmacy-territory-updates && ddev drush @pharmacy.local updatedb -y && ddev drush @pharmacy.local cim -y`
Check that the update hook runs cleanly. Edit the person again, and see that there are no errors, and the international option is no longer selected (nor available)
